### ISSUE TITLE
Questions about the training set of supervisied finetuning 

### ISSUE THREAD

**Issue by LightersWang** [tsunghan-wu/D2ADA#3]
Hello there,

Thanks for sharing such a well-organized repo, it'll be very helpful for the active learning community.

However, I have some questions about the training set of supervised finetuning.

In [line 45 of `dataloader/active_dataset.py`](https://github.com/tsunghan-wu/D2ADA/blob/6738595338e5124f8797c7b186d2afdba0ef5257/dataloader/active_dataset.py#L45), why the training set of iteration 1 is still the source labeled set? Since in the end of the iteration 0, we have already queried some samples for target labeled set.

What's more, in [RegionActiveDataset](https://github.com/tsunghan-wu/D2ADA/blob/main/dataloader/region_active_dataset.py), the training set of supervised finetuning for every iteration is  the concat of source labeled set and target labeled set. I'm confused of the inconsistency between ActiveDataset and RegionActiveDataset.

Thank you, and I look forward to hearing from you.

**Comment by tsunghan-wu** [comment]
Hi @LightersWang,

Thank you for your valuable feedback. 

1. I apologize for the incorrect use of `iteration == 1` in the repo. In our private repo, the first iteration was used for training on the source domain dataset only, while the 2nd to 6th iterations were used for training with 1% to 5% target labels. To align with our paper, we plan to change it to 0-5 in the public version. I have corrected this mistake in [utils/common.py](https://github.com/tsunghan-wu/D2ADA/blob/main/utils/common.py#L155-L157) but overlooked a similar issue in line 45 of [dataloader/active_dataset.py](https://github.com/tsunghan-wu/D2ADA/blob/main/dataloader/active_dataset.py#L45). Thank you for bringing this to our attention, it has now been corrected. **Note that this error does not affect the reproduction of our experimental results**, because our region-based active learning method does not use this file at all (see the next question for details).

2. In this work, we primarily use "pre-divided regions" as the fundamental labeling unit and set `--active_mode=region` as the default setting ([ref](https://github.com/tsunghan-wu/D2ADA/blob/main/utils/common.py#L145-L146)) as we all know that region-based labeling is much more efficient than image-based one. This requires a [RegionActiveDataset](https://github.com/tsunghan-wu/D2ADA/blob/main/dataloader/region_active_dataset.py) to manage the datasets with a more complicated region-level "expand_training_set" method. Additionally, we offer the option `--active_mode=scan` for image-based active learning, although we did not provide any experimental results for this in our paper. In the "ActiveDataset," we only maintain the `im_idx` instead of the additional `suppix`.

Thank you again for your great catch. I hope this clarification resolves your concerns :-)
